EapImport/src/main/java/de/hpi/unicorn/importer/json/JsonImporter.java

```diff
@@ -95,6 +95,40 @@ public static boolean generateAttributeDependenciesFromString(String dependencie
         return true;
     }
 
+    public static Map<Object,Object> generateValuesFromString(String valuesString) {
+        Map<Object,Object> result = new HashMap<>();

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') ',' is not followed by whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AWhitespace_After_BP2016)